### PR TITLE
Support node-search context menu in iterators

### LINKED ISSUE #1353: Right click in *iterator space not works
Can't create nodes by right click popup because it not works.

### PR DESCRIPTION
**Pull request by joeyballentine**
Closes #1353

### CODE REVIEW

File: src/renderer/hooks/usePaneNodeSearchMenu.tsx [hunk 1]
```diff
@@ -379,7 +380,7 @@ export const usePaneNodeSearchMenu = (
                     },
                     nodeType: schema.nodeType,
                 },
-                stoppedOnIterator || undefined
+                stoppedOnIterator || parent || undefined
```
Comment (RunDevelopment) [inline src/renderer/hooks/usePaneNodeSearchMenu.tsx:383]: The `|| undefined` is unnecessary, I think.

Comment (joeyballentine) [reply src/renderer/hooks/usePaneNodeSearchMenu.tsx:383]: Since it was there before I'd rather not remove that unless you're sure

Comment (RunDevelopment) [reply src/renderer/hooks/usePaneNodeSearchMenu.tsx:383]: I am sure. The reason it was there in the first place is that `stoppedOnIterator` allows `null` (for some reason), but the parameter expected `string | undefined`.

Comment (RunDevelopment) [reply src/renderer/hooks/usePaneNodeSearchMenu.tsx:383]: Also, with `null` and `undefined` in general: TypeScript is really good at tracking these. If TS says it's fine, then it most likely is.

Comment (joeyballentine) [inline src/renderer/hooks/usePaneNodeSearchMenu.tsx:383]:
```suggestion
                stoppedOnIterator || parent
```

**Review by RunDevelopment — APPROVED** [review]

**Pull request merged**